rel-eng/custom/custom.py

```diff
@@ -195,6 +197,18 @@ def _fetch_local(self):
 
         return gitrev
 
+    def _fetch_prebuilt_local(self):
+        source_files = [os.path.expanduser(source_file) for source_file in self.builder.args['source_files']]
+        dest_dir = os.path.join(self.builder.rpmbuild_sourcedir, 'archive')
+        if not os.path.exists(dest_dir):
+            os.mkdir(dest_dir)
+
+        for src in source_files:
+            debug("Copying %s to %s" % (src, os.path.join(dest_dir, '')))
+            shutil.copy(src, os.path.join(dest_dir, ''))
```

Comment:
I'd prefer to be explicit and use `os.path.basename(srcfile)` instead of an empty string.